modules/ROOT/pages/design-mocking-service.adoc

```diff
@@ -3,16 +3,21 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-Use the mocking service to provide a public link to the API you are designing. The mocking service returns the responses (both HTTP status codes and example payloads) that are defined in your API specification, and is valuable for testing or for simply exploring how your API behaves.
+Use the mocking service to provide a public link to the API you are designing. The mocking service returns the responses (both HTTP status codes and example payloads) that are defined in your API specification. The mocking service is valuable for testing or for simply exploring how your API behaves.
 
 == Before You Begin
 
 You must be working in a project to create an API specification in API Designer.
 
 == About this task
+
 Simulating calls to the API is a critical design task that helps you troubleshoot problems and demo the API to prospective users even before you have implemented it. You can simulate calls to the API in API Designer before publishing the API specification or in Exchange after publishing the API specification there.
 
-When you activate the mocking service, it exposes a mock service that you can use for testing in lieu of an actual implementation. If you have already specified a base URI (RAML) or base path (OAS), the mocking service comments it out. When you deactivate the mocking service, it removes the base URI or base path that it used and uncomments your base URI or base path. For example, if the base URI in a RAML API specification is `+http://jsonplaceholder.typicode.com+`, the mocking service replaces it temporarily like this, where `+https://anypoint.mulesoft.com/mocking/api/v1/links/+`, etc. is the URI used by the mocking service:
+When enabled, the mocking service provides a link you can use to test the API. You can share this link with other Anypoint Platform users or with external users who need to see how the API behaves.
```

Comment:
This should be changed the Mocking Services has no longer the toggle button:
<img width="407" alt="2020-10-07_09-03-58" src="https://user-images.githubusercontent.com/12468128/95328824-6cd81700-087c-11eb-8d30-00033e12ed51.png">

Now has a selector (only when a baseUri defined):
<img width="878" alt="2020-10-07_09-10-31" src="https://user-images.githubusercontent.com/12468128/95329238-056e9700-087d-11eb-8b1c-4ad3038612cc.png">

If a baseUri is not defined, nothing is shown:
<img width="429" alt="2020-10-07_09-11-49" src="https://user-images.githubusercontent.com/12468128/95329338-2afba080-087d-11eb-9be1-314cd57c312f.png">

Reply:
In addition to what Ignacio said, with the new implementation, Mocking Service is always on, meaning that users don't need to explicitly enable it but the mock instances are created on-demand. As a result, we decided to remove the mocking Service switch and to auto-populate the documentation panel with Mocking Service as a server (this last part is what Ignacio showed with the screenshots)

Reply:
On the other hand, there was a change regarding the accessibility of the Mocking Service. In the past, when enabling Mocking Service, the API Designer used to generate a URL that users could share with anyone since it was public (it didn't require authentication), but now, the Mocking Service URL is private by default (it does require authentication to be used). So, in order to give the users the chance to keep sharing the Mocking Service URL, we added a new panel where users can create a public link and share it